### ISSUE TITLE
Accessibility review watch party

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#2610]
Gather the whole team together to go over key insights from the unauthenticated form-building tests conducted by Fable

**Comment by srtalbot** [comment]
Moving to end of Nov when we have some time for usability and accessibility debt.

**Comment by Abi-Nada** [comment]
Closing this card as highlights were shared via Slack and we'll be engaging in another round of testing in January. @connorscarolyns feel free to flag if there are any leftover items from this testing that hasn't been documented into cards yet
